php/fizzbuzz.php

```diff
@@ -0,0 +1,8 @@
+<?
+foreach (range(1, 100) as $i) {
+    if ($i % 15 === 0): echo "FizzBuzz" . PHP_EOL;
+    elseif ($i % 3 === 0): echo "Fizz" . PHP_EOL;
+    elseif ($i % 5 === 0): echo "Buzz" . PHP_EOL;
+    else: echo $i . PHP_EOL;
+    endif;
+}
```

Comment:
:O is the closing brace optional?

Reply:
you mean `?>` to close the php tag? It's not even optional, but discouraged by [PSR-2](https://www.php-fig.org/psr/psr-2/), not sure about the reasons though, haven't played with PHP for a few years